frontend/src/modules/community-member/components/community-member-platform-tabs.vue

```diff
@@ -109,6 +123,15 @@ export default {
     &--active {
```

Comment:
Again, if we won't be using the active class anymore, let's just remove it 👍

Reply:
It is still being used by the `All Platforms` button.